staging_service/app.py

```diff
@@ -108,31 +104,45 @@ def _file_type_resolver(path: PathPy) -> FileTypeResolution:
         return FileTypeResolution(unsupported_type=ext)
 
 
+def _make_dts_file_resolver() -> Callable[[Path], FileTypeResolution]:
+    """Makes a DTS file resolver."""
+
+    def dts_file_resolver(_: PathPy):
+        return FileTypeResolution(parser=parse_dts_manifest)
+
+    return dts_file_resolver
```

Comment:
Should this return a FTR with an unsupported type if the file extension isn't JSON?

Reply:
This one I went back and forth on. Any file can have any extension, right? I get that we're using that convention to assume that csv and xls files are what they say they are, but they don't have to. I felt that explicitly saying that, with the `dts` flag, ALL files will be treated as manifests would be enough to skip the extension check.

The other side of that is JSON is usually in `.json` files. And much of the rest of the codebase sticks to that convention.

IOW, I'll add it to keep the code coherent, but I don't feel like it matters.

Reply:
:+1: